tips/TIP-0046/tip-0046.md

```diff
@@ -450,116 +437,90 @@ Let the work score of a _Basic Block_ be defined as follows.
-        </blockquote>
-        <table>
-          <tr>
-            <th>Name</th>
-            <th>Type</th>
-            <th>Description</th>
-          </tr>
-          <tr>
-            <td>Block ID</td>
-            <td>ByteArray[36]</td>
-            <td>The Block ID of the strong parent.</td>
-          </tr>
-        </table>
+        <blockquote>A reference to a block. Defined in <a href='../TIP-0046/tip-0046.md#parent'>TIP-46 (Parent)</a>.</blockquote>
       </details>
     </td>
   </tr>
-  <tr>
   <tr>
     <td>Weak Parents Count</td>
     <td>uint8</td>
-    <td>The number of blocks that are weakly directly approved.</td>
+    <td>The number of blocks following, which are weakly directly approved.</td>
   </tr>
   <tr>
-    <td>Weak Parents <code>anyOf</code></td>
+    <td valign="top">Weak Parents <code>optAnyOf</code></td>
     <td colspan="2">
       <details>
         <summary>Parent</summary>
-        <blockquote>
-          References another directly approved block.
-        </blockquote>
-        <table>
-          <tr>
-            <th>Name</th>
-            <th>Type</th>
-            <th>Description</th>
-          </tr>
-          <tr>
-            <td>Block ID</td>
-            <td>ByteArray[36]</td>
-            <td>The Block ID of the parent.</td>
-          </tr>
-        </table>
+        <blockquote>A reference to a block. Defined in <a href='../TIP-0046/tip-0046.md#parent'>TIP-46 (Parent)</a>.</blockquote>
       </details>
     </td>
   </tr>
-  <tr>
   <tr>
     <td>Shallow Like Parents Count</td>
     <td>uint8</td>
-    <td>The number of blocks that are directly referenced to adjust opinion.</td>
+    <td>The number of blocks following, which are directly referenced to adjust opinion.</td>
   </tr>
   <tr>
-    <td>Shallow Like Parents <code>anyOf</code></td>
+    <td valign="top">Shallow Like Parents <code>optAnyOf</code></td>
     <td colspan="2">
       <details>
         <summary>Parent</summary>
-        <table>
-          <tr>
-            <th>Name</th>
-            <th>Type</th>
-            <th>Description</th>
-          </tr>
-          <tr>
-            <td>Block ID</td>
-            <td>ByteArray[36]</td>
-            <td>The Block ID of the parent.</td>
-          </tr>
-        </table>
+        <blockquote>A reference to a block. Defined in <a href='../TIP-0046/tip-0046.md#parent'>TIP-46 (Parent)</a>.</blockquote>
       </details>
     </td>
   </tr>
+  <tr>
+    <td>Payload Length</td>
+    <td>uint32</td>
+    <td>The length of the following payload in bytes. A length of 0 means no payload will be attached.</td>
+  </tr>
   <tr>
     <td>Highest Supported Version</td>
     <td>uint8</td>
-    <td>The highest supported protocol version the issuer of this block supports. TODO: Add more details.</td>
+    <td>The highest supported protocol version the issuer of this block supports.</td>
   </tr>
   <tr>
     <td>Protocol Parameters Hash</td>
-    <td>ByteArray[32]</td>
+    <td>uint8</td>
```

Comment:
This seems weird

Reply:
Copy-Paste error. Fixed it. Thanks :+1:.